.github/workflows-source/release-testing.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           run: |
             bazel test \
               --config=stamped \
-              --test_tag_filters=system_test_nightly,system_test_staging,system_test_hotfix \
+              --test_tag_filters=system_test_nightly \
```

Comment:
Considering your [earlier comment](https://github.com/dfinity/ic/pull/5611#pullrequestreview-2943284553) about not naming tagging targets on *when* they run, what about renaming this to `system_test_release`. That would also make the naming between the job and the tags consistent which will help with understanding and grepping.

Reply:
Makes a ton of sense!

Reply:
`large_system_test` could also be an option. I'll let you think about it.